.github/workflows/build.yml

```diff
@@ -91,7 +91,7 @@ jobs:
       - run: |
           cd python-bindings
           pip install -r requirements-dev.txt
-          pip install transformers==4.12.3
+          pip install transformers==4.22.2
```

Comment:
You will want 4.21 (4.22 the download utilities used in the test and would have to be updated in a separate PR)

Reply:
Yes, you're right, thank you very much!

Now it's failing during some of the `prophetnet` tests execution.

Reply:
Yes you'll want 
```python
            output_baseline.append(self.base_tokenizer.encode_plus(example.text_a,
                                                                   add_special_tokens=True,
                                                                   return_overflowing_tokens=True,
                                                                   return_special_tokens_mask=True,
                                                                   return_token_type_ids=True,
                                                                   max_length=128))
```
at https://github.com/guillaume-be/rust-tokenizers/blob/2913e21686446c093e8b216d83f0d4e647fdb839/python-bindings/tests/test_tokenization_sst2.py#L557

Reply:
Pipeline fixed, thanx.